.github/actions/setup-kind/action.yml

```
@@ -32,13 +33,21 @@ inputs:
       The timeout for waiting for the ingress controller to become ready.
       Default is 90 seconds, which should be enough for most cases.
     default: 90s
+  kind-config-override:
+    description: |
+      Optional YAML string to override the default kind configuration passed to
+      kind during cluster creation.
+      See https://kind.sigs.k8s.io/docs/user/configuration/ for details.
+    default: ""
 runs:
   using: "composite"
   steps:
     - name: Create cluster
+      env:
+        KIND_CFG: ${{ inputs.kind-config-override == '' && format('{0}/kind.yml', github.action_path) || inputs.kind-config-override }}
```

weren't empty strings already falsy?


```suggestion
        KIND_CFG: ${{ inputs.kind-config-override || format('{0}/kind.yml', github.action_path) }}
```